application/error_handlers.py

```diff
@@ -0,0 +1,27 @@
+import flask
+from flask import jsonify
+from requests import HTTPError
+
+import logging
+from structlog import wrap_logger
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+blueprint = flask.Blueprint('error_handlers', __name__)
+
+
+@blueprint.app_errorhandler(Exception)
+def exception_error(error):
+    logger.error("Unhandled Exception: ", error)
```

Comment:
logger.exception might be more suitable here after a discussion with @JamesGardiner the other day

Reply:
Agreed - `logger.exception` will log the stack trace automatically - bonus points if you use `structlog` to get the exception into a well-structured key:val pair ingestible by splunk